<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstly</class>
 <widget class="QMainWindow" name="firstly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stud">
     <attribute name="title">
      <string>Студент</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>681</width>
        <height>281</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="auth_log">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nick_auth">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="auth">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>200</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: white;
font: 75 8pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QPushButton:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QPushButton:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
        </property>
        <property name="text">
         <string>Авторизоваться</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="auth_log_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nick_auth_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>140</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QPushButton" name="auth_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>200</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: white;
font: 75 8pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QPushButton:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QPushButton:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
        </property>
        <property name="text">
         <string>Зарегестрироваться</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="_error">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="_combo_stud">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{color: white;
font: 75 9pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QComboBox:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QComboBox:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
      </property>
      <item>
       <property name="text">
        <string>Авторизация</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Регистрация</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="maneg">
     <attribute name="title">
      <string>Меннеджер</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>681</width>
        <height>291</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="auth_log_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nick_auth_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="auth_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>240</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: white;
font: 75 8pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QPushButton:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QPushButton:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
        </property>
        <property name="text">
         <string>Зарегестрироваться</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_log_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_7">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>200</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_8">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_log_6">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_9">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>120</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_log_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>День рождения</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_10">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_7">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Ключ</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="auth_log_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="_auth_maneg">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>140</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pass_auth_6">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>461</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid black;
    border-radius: 8px;
    padding: 2px;
    outline: none;
	font: 12pt &quot;Times New Roman&quot;;
    margin-left: auto;
    margin-right: auto;
    width: 6em
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="auth_pass_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: solid black;</string>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
       <widget class="QPushButton" name="auth_4">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>210</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: white;
font: 75 8pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QPushButton:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QPushButton:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
        </property>
        <property name="text">
         <string>Авторизоваться</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="_error_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="_combo_maneg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{color: white;
font: 75 9pt &quot;Comic Sans MS&quot;;
background-color: rgb(0, 0, 0);
border: 3px;
border-radius: 10px;
}
QComboBox:hover{ background-color: lightgray; padding: 5px;
	border-style: outset; border: 1px solid gray; font-weight: bold;}
QComboBox:pressed {border: 2px solid gray; padding: 5px;
    background-color: silver; border-style: inset; font-weight: bold;}</string>
      </property>
      <item>
       <property name="text">
        <string>Регистрация</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Авторизация</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_combo_stud</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_combo_maneg</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget_2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
